script.js

```diff
@@ -1 +1,19 @@
 // script.js
+
+function getTriangleArea (a, h) {
+		if ((a <= 0) || (h <= 0)) {
+			console.log("Nieprawidłowe dane");
```

Comment:
what should be returned here?

Reply:
return false?

Reply:
or message that data are wrong. :) e.g.:

`return 'Wrong data';`